doc/new_boilerplate_migration.md

```diff
@@ -986,6 +986,8 @@ UiFactory<FooProps> Foo = uiForwardRef(
 
 ## Upgrading Existing Code
 
+__First, you must upgrade your components to `UiComponent2`__. Check out the [`UiComponent2` Migration Guide](ui_component2_transition.md) to learn about the benefits of `UiComponent2`, the codemod script you can run, and other updates you may need to make manually.
```

Comment:
The codemod script was removed as part of the null safety migration for `over_react_codemod`.

Reply:
Ah, right; I'll update this with docs on how to use that from an older version